README.md

```diff
@@ -58,6 +58,21 @@ jobs:
           cljstyle: 0.15.0             # cljstyle
           cmd-exe-workaround: 'latest' # Replaces `clojure` with `deps.clj` on Windows
           zprint: 1.2.3                # zprint
+          
+      # Optional step:
+      - name: Cache clojure dependencies
+        uses: actions/cache@v3
+        with:
+          path: |
+            ~/.m2/repository
+            ~/.gitlibs
+            ~/.deps.clj
+          # List all files containing dependencies:
+          key: cljdeps-${{ hashFiles('deps.edn') }}
```

Comment:
I've seen variation that use `runner.os` as the part of key, but seems like it's quite safe to use same cache for all platforms. Maybe I'm wrong, though.

Reply:
Agree, I don't see the reason to keep `runner.os`. Let's keep it the way you brought it :) thanks!